pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -2739,6 +2739,11 @@
       "AssemblyVersionInPackageVersion": {
         "4.0.0.0": "4.4.0"
       }
+    },
+    "System.Net.Http.Listener": {
```

Comment:
I think the contract should be called "System.Net.HttpListener".  It shouldn't have a "." between "Http" and "Listener".